opal/mca/common/ofi/common_ofi.c

```diff
@@ -772,8 +772,8 @@ struct fi_info *opal_common_ofi_select_provider(struct fi_info *provider_list,
     pmix_value_t *pmix_val;
     struct fi_pci_attr pci;
     int num_distances = 0;
-    bool near;
 #endif
+    bool near = false;
```

Comment:
Just curious: why are you moving this out of an #if block?  That wouldn't seem to fix a coverity uninitialized error.  Was there some other compile error, too?  If so, please update the commit message to match.